inspectit-oce-core/build.gradle

```diff
@@ -54,6 +55,12 @@ dependencies {
             "io.prometheus:simpleclient_httpserver:${prometheusClientVersion}"
     )
 
+    testImplementation(
+            'org.springframework:spring-test:5.1.3.RELEASE',
+            'org.apache.httpcomponents:httpclient:4.5.6',
+            'org.mockito:mockito-core:2.23.4'
```

Comment:
can you add `assertj` as well, you will like it :)

http://joel-costigliola.github.io/assertj/